libr/core/cbin.c

```diff
@@ -3035,13 +3035,14 @@ static void bin_pe_resources(RCore *r, int mode) {
 		} else if (IS_MODE_JSON (mode)) {
 			r_cons_printf("%s{\"name\":%d,\"index\":%d, \"type\":\"%s\","
 					"\"vaddr\":%"PFMT32d", \"size\":%d, \"lang\":\"%s\"}",
-					index? ",": "", name, index, type, vaddr, size, lang);
+					index? ",": "", name, index, type, (ut32)vaddr, size, lang);
+
 		} else {
 			char *humanSize = r_num_units (NULL, size);
 			r_cons_printf ("Resource %d\n", index);
```

Comment:
W t f who the hell wrote that crap????? Remove all those \t and use 1or 2 spaces